internal/setup/setup.go

```diff
@@ -623,3 +650,75 @@ func Select(release *Release, slices []SliceKey) (*Selection, error) {
 
 	return selection, nil
 }
+
+// convertPathInfoToYamlPath converts a PathInfo object to a yamlPath object.
+func convertPathInfoToYamlPath(pi *PathInfo) (*yamlPath, error) {
```

Comment:
This name is a bit of a mouthful. How about `pathInfoToYAML`? (Note also the uppercase `YAML` which is Go style for acronyms.)

Reply:
That's a better name, thanks!